shifthelper/checks.py

```diff
@@ -2,6 +2,7 @@
 from . import retry_smart_fact_crawler as sfc
 from .tools.is_shift import is_shift_at_the_moment
 import requests
+from requests.exceptions import RequestException
```

Comment:
All requests exceptions inherit from `RequestException`, so I took this one to be safe.